app/views/layout.erb

```diff
@@ -15,6 +15,7 @@
 		<!-- Jquery -->
 		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js">
 		</script>
+		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
We need to decide which icons we are going to use in our project! Bootstraps or Materialize icons!
We already have bootstrap icons(glyphicons) in some parts of our code. Let's choose one of them and stick with it!